frontend/src/utils/use_media_query.ts

```diff
@@ -0,0 +1,10 @@
+import { useEffect, useState } from 'react';
+
+export function useMediaQuery(mediaQuery: string) {
```

Comment:
why do we do this instead of using the lib? -> provide rationale in the commit message plz

Reply:
Done